content/docs/aws/authentication-oidc.md

```diff
@@ -0,0 +1,129 @@
++++
+title = "Authentication and Authorization through OIDC"
+description = "Authentication and authorization support through OIDC for Kubeflow in AWS"
+weight = 90
++++
+
+This section shows the how to setup Kubeflow with authentication and authorization support through OIDC in Amazon Web Services (AWS). Similar to [Cognito tutorial](https://www.kubeflow.org/docs/aws/authentication/).
+
+## Enable TLS and Authentication
+[OIDC](https://openid.net/connect/) allows clients to verify the identity of the End-User based on the authentication performed by an Authorization Server. OIDC allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. For this tutorial, we take Github Login as an example.
+
+In order to authenticate and manage users for Kubeflow, let's first connect Auth0 app to Github. You can follow instructions [here](https://auth0.com/docs/connections/social/github).
```

Comment:
You probably want to address `Auth0` and `Github` is the solution you pick for the tutorial, user can use any OIDC compatible solutions.

Reply:
Sure, users are free to use any OIDC compatible solutions. And I mentioned "we take Github Login as an example". And thank you for your reminding, I'll add some words in the doc.